lib/components/narrative/default/default-itinerary.js

```diff
@@ -109,22 +112,20 @@ const ITINERARY_ATTRIBUTES = [
     render: (itinerary, options) => {
       const formattedStartTime = itinerary.startTime + options.offset
       const formattedEndTime = itinerary.endTime + options.offset
```

Comment:
Rename these vars to `startTimeWihOffset`, `endTimeWithOffset`.

Reply:
Renamed in 2dd6711fa9cb06428815df5ab9d1515fe95c1826